### PR TITLE
File conversion for zarr with variable corrections + MELCC + CanSWE + Packaging improvements

### LINKED ISSUE #44: Split rstdmf module functions into a different library
The `rstdmf` utilities are command line processes, custom-made by Ouranos, and are solely used by 2-3 people in the company. These utilities should be split off into a few examples in `scriptes-internes`, or made into an optional module only installable from git.

At the very least, any package produced for PyPI should exclude this module from the MANIFEST.

### LINKED ISSUE #86: Some daily CORDEX variables given at midnight
I haven't dug everywhere, but at least for `simulation/raw/CMIP5/CORDEX/NAM-44/UQAM/UQAM-CRCM5/CCCma-CanESM2/historical/r1i1p1/day/`, `tasmin` and `tasmax` are given at noon, while `pr` is given at midnight.

This fails `xscen`.  In  `extract_dataset`, we call  `to_dataset_dict` which opens all variables of a given frequency into the same dataset, without checking for this kind of issue. We do have a `ensure_correct_time` check, but it happens afterwards, when the duplicated time axis was already created by xarray. And so the test fails.

@RondeauG @juliettelavoie @Zeitsperre 
Is this an xscen bug or a data bug?

A solution in xscen would be to remove the variable union aggregation from the catalog esm spec and iterate over the variables individually. We already iterate over them, but we use the merged dataset as created by intake-esm.

EDIT: Another solution would to put the "ensure_correct_time" test and correction in the `preprocess` of `to_dataset_dict`, At least, that's going to be my patch fix for the moment.

### LINKED ISSUE #91: ERA5/-Land evaporation and potential evaporation are negative up
These two variables are negative when going up. A sign change is needed to be consistent with climate model variable standards.

### PR DESCRIPTION
**Pull request by Zeitsperre**
Fixes #91
Fixes #86
Fixes #44

### CODE REVIEW

File: miranda/structure/data/ouranos_schema.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    value: simulation
+    structure:
+      - type
+      - processing_level
+      - mip_era
+      - activity
+      - option: processing_level
+        value: raw
+        is_true: processing_level
+        else: bias_adjust_project
```
Comment (aulemahal) [inline miranda/structure/data/ouranos_schema.yml:51]:
```suggestion
     - option: processing_level
        value: biasadjusted
        is_true: bias_adjust_project
      - mip_era
      - activity
```
As suggested in slack.

Comment (Zeitsperre) [reply miranda/structure/data/ouranos_schema.yml:51]: @juliettelavoie @tlogan2000 @RondeauG How do we feel about this?

Comment (juliettelavoie) [reply miranda/structure/data/ouranos_schema.yml:51]: ok with me!

Comment (RondeauG) [reply miranda/structure/data/ouranos_schema.yml:51]: Good for me too

Comment (Zeitsperre) [reply miranda/structure/data/ouranos_schema.yml:51]: Alrighty

**Pull request merged**